aten/src/ATen/native/TensorShape.cpp

```diff
@@ -8,6 +8,25 @@
 namespace at {
 namespace native {
 
+static void check_cat_no_zero_dim(TensorList tensors) {
+  for(size_t i = 0; i < tensors.size(); ++i) {
```

Comment:
If you don't need `i` you can just use a for-each loop:

```
for (auto& t : tensors) {
  ...
}
```

Reply:
oh, I was going to add a position in the error message, let me do that.